Description: Ignore the osgi-test modules (maven-debian-helper has some troubles ignoring nested modules)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/jaxb-ri/pom.xml
+++ b/jaxb-ri/pom.xml
@@ -705,7 +705,6 @@
             </properties>-->
             <modules>
                 <module>docs</module>
-                <module>tools/osgi_tests</module>
             </modules>
         </profile>
         <profile>
